tools/cp-caps/rcp_caps_test.py

```diff
@@ -100,11 +101,23 @@ class Frame:
         Attributes:
           name: The description of the frame.
           tx_frame: The psdu of the frame that to be sent.
-          dst_address: The destination Mac address of the tx_frame.
+          dst_address: The destination Mac address of the tx_frame. It is used by the receiver to filter
+              out the tx_frame.
+          is_security_processed: The value of the otRadioFrame.mInfo.mTxInfo.mIsSecurityProcessed field.
+              If it is set to False, the active_dataset and src_ext_address are also should be set for the
+              radio driver to encrypt the tx_frame.
+          expect_rx_frame: The frame expected to be received. The frame expected to be received should be
+              the same with the tx_frame if the expect_rx_frame is set to None.
+          active_dataset: The active dataset.
+          src_ext_address: The source extended Mac address of the transmitter.
         """
         name: str
         tx_frame: str
         dst_address: str
+        is_security_processed: Optional[bool] = True
```

Comment:
Why is it true by default?

Reply:
Because the radio driver will try to encrypt the frame if the frame contains a security header and the is_security_processed is set to false. The is_security_processed is set to True by default to prevent the radio driver from accidentally changing the frame.